conftest.py

```diff
@@ -0,0 +1,27 @@
+import pytest
+import requests
+from selenium import webdriver
+
+
+def pytest_addoption(parser):
+    parser.addoption("--browser", default="chrome")
+
+
+@pytest.fixture(scope="session", params=["chrome", "firefox", "opera", "safari"])
+def browser(request):
+    driver_path = "/Users/Ilya/Documents/drivers"
```

Comment:
I would do it as parameter with default value, so that anyone could run this code without changing this value but pass it as --drivers_folder for example